cmd/server/server.go

```diff
@@ -40,7 +40,17 @@ func runProxy(d driver.Driver, n *negroni.Negroni, logger *logrus.Logger) func()
 			Transport: proxy,
 		}
 
-		n.Use(negronilogrus.NewMiddlewareFromLogger(logger, "oathkeeper-proxy"))
+		proxyLogger := negronilogrus.NewMiddlewareFromLogger(logger, "oathkeeper-proxy")
```

Comment:
We are using a small helper for this in other projects:

```
github.com/ory/x/reqlog.NewMiddlewareFromLogger(logger, "oathkeeper-api").ExcludePaths(healthx.ReadyCheckPath, healthx.AliveCheckPath)
```

Which basically moves the health checks to "debug" output. It's tested etc so I would prefer using that here :)